spec/request/documents_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe 'Documents management', type: :request do
+  it 'should fail if no file is provided' do
+    post '/v0/claims/1/documents'
+    expect(response).to_not be_success
```

Comment:
can you make a more precise assertion here? glancing at this, it isn't clear that this spec is testing a failure in our endpoint vs. a failure in the evss request. this may be better as a controller test so you can assert the raise exception rather than the response it generates